plugins/ks.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
```

Comment:
Do you have any affiliation to KubeSphere project, or are you just a user?

Reply:
I'm an employee of KubeSphere. But this project is not an official project yet. So I just put it under my personal account for now.

Reply:
> I'm an employee of KubeSphere. But this project is not an official project yet. So I just put it under my personal account for now.

If it's not an official plugin, it wouldn't be proper to assign this name to your personal repo. See naming guidelines.

Reply:
I totally understand your concern. But it's a perfect name in my view. This plugin might become the official one in the future. Given I cannot make a promise on that, I probably have no choice. I'm going to close this PR until something changes in the future. Anyway, thanks for your review.